packages/calcite-components/src/components/popover/popover.e2e.ts

```diff
@@ -699,4 +701,63 @@ describe("calcite-popover", () => {
       });
     });
   });
+
+  describe("theme", () => {
+    const popoverHTML = html`
+      <calcite-popover heading="I'm a heading in the header using the 'heading' prop!" closable>
+        Lorem Ipsum
+      </calcite-popover>
+    `;
+
+    describe("default", () => {
+      themed(popoverHTML, {
```

Comment:
Can you inline the popover HTML? It’s used for two tests with slight differences in each, where the default configuration should not be closable.